bigquery_etl/cli/query.py

```diff
@@ -626,14 +627,14 @@ def _backfill_query(
     "--checks/--no-checks", help="Whether to run checks during backfill", default=False
 )
 @click.option(
-    "--scheduling_parameters_override",
-    "--scheduling-parameters-override",
-    "-spo",
+    "--scheduling_overrides",
+    "--scheduling-overrides",
+    "-so",
```

Comment:
I'd suggest removing the shortened `-so` form as it's kind of non-standard for Unixy CLIs (usually single-dashed options are one character, and such options that are flags can be specified with a single dash followed by multiple characters, like `-so` instead of `-s -o`).

Reply:
removed